core/cloudflow-akka/src/main/resources/cassandrapersistence.conf

```diff
@@ -0,0 +1,45 @@
+# Adopted from https://github.com/akka/akka-persistence-cassandra/blob/master/example/src/main/resources/common.conf
+# maybe https://doc.akka.io/docs/akka-persistence-cassandra/current/configuration.html
+akka {
+  persistence {
+    journal {
+      plugin = "akka.persistence.cassandra.journal"
+      # to create the schema
+      auto-start-journals = ["akka.persistence.cassandra.journal"]
+    }
+    snapshot-store {
+      plugin = "akka.persistence.cassandra.snapshot"
+    }
+
+    cassandra {
+      journal {
+        class = "akka.persistence.cassandra.journal.CassandraJournal"
+        keyspace-autocreate = true
```

Comment:
Would users do this in prod?

Reply:
They can overwrite it, but... The way it is implemented, it create if does not exist. So its ok to do this.